js/gsbn.js

```diff
@@ -21,7 +21,7 @@ var gsbn = {
               "%D7%94%D7%A2%D7%A8%D7%95%D7%AA%20%D7%93%D7%95%D7%91%D7%A8"  // Hebrew
 	    ]
       
-      if (i18n.indexOf(encodeURI(window.top.document.body.getAttribute("aria-label").toLowerCase())) == -1) {
+      if (!window.top.location.pathname.startsWith('/presentation/') && !window.top.location.pathname.endsWith('/present')) {
```

Comment:
shouldn't this be an "or" instead of an "and"? We want to ensure that both the conditions are met, not just one of them, right?